.github/workflows/vulnerabilities.yml

```diff
@@ -21,6 +21,10 @@ jobs:
       run:
         working-directory: .
     steps:
+    - id: string
```

Comment:
What is the nature of this update?  Should this be in another PR?

Reply:
This is related to the comment above. because I was working on a fork which has a capital letter in the repo name, it was failing to build the docker image in this github action. I don't think it's a great idea for this particular case, because it relies on a third party github action for security vulnerability checks, so I'll yeet it. :slightly_smiling_face: